base/base/static_for.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <type_traits>
+#include <utility>
+#include "FnTraits.h"
+#include "CTArray.h"
+
+namespace detail
+{
+template <class T, T Begin, T... Is>
+constexpr bool static_for(auto && f, std::integer_sequence<decltype(Begin), Is...>)
+{
+    return (
+        std::forward<decltype(f)>(f)(
```

Comment:
Why do you need `std::forward` here?

Reply:
The "always forward arguments while taking an &&" rule.

Reply:
`std::forward` is supposed to be used only for passing function arguments.

Reply:
Ok, I'll remove it here.